lms/static/js/learner_dashboard/models/course_entitlement_model.js

```diff
@@ -1,23 +1,20 @@
+import Backbone from 'backbone';
+
 /**
  *  Store data for the current entitlement.
  */
-(function(define) {
-    'use strict';
+class CourseEntitlementModel extends Backbone.Model {
+  constructor(attrs, ...args) {
```

Comment:
Does this take/need ...args? The old model doesn't seem to.

Reply:
The old model didn't override the constructor at all. It just used the fact that es5 let you override/define class fields directly, in a way that took effect before the constructor even ran. es6 is stricter than that. Hence the pattern I use of supplementing constructor dictionary arguments with defaults.

In this case, I probably got fancier than I needed to be. The Backbone.Model constructor takes two arguments and I didn't care about the second, only the first. I could have just used "attrs, options" as my args and passed options in unmodified. But I was imagining a future in which Backbone.Model grows a third argument. ::shrug::

Reply:
Id be surprised if Backbone models grew a third argument after not being touched since 2016, but if it works I see no reason to change it out to options. 👍